CHANGELOG.md

```diff
@@ -24,6 +24,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Tooltips for long names on cards (projects, tasks, cloud storages, and models)
   (<https://github.com/opencv/cvat/pull/7550>)
 
+- Add devcontainer configuration for local system as well as for GitHub Codespaces
```

Comment:
You should not modify the file directly. Please read comments above

Reply:
Thank you @nmanovic. I missed to refer that. I have now corrected the issue.